.gitattributes

```diff
@@ -0,0 +1,6 @@
+*.proto		text
```

Comment:
spaces vs tabs for some of these?

Reply:
gitattributes won't deal with indentation, only line endings.  you'd need a pre-commit hook against some user tool